mailbox/server.go

```diff
@@ -85,23 +85,13 @@ func (s *Server) Accept() (net.Conn, error) {
 			s.ctx, s.serverHost, s.client, receiveSID, sendSID,
 		)
 		if err != nil {
-			log.Errorf("couldn't create new server: %v", err)
-			if err := mailboxConn.Close(); err != nil {
-				return nil, &temporaryError{err}
-			}
 			return nil, &temporaryError{err}
 		}
 		s.mailboxConn = mailboxConn
 
 	} else {
 		mailboxConn, err := RefreshServerConn(s.mailboxConn)
 		if err != nil {
-			log.Errorf("couldn't refresh server: %v", err)
-			if err := mailboxConn.Stop(); err != nil {
-				return nil, &temporaryError{err}
```

Comment:
yeah im not sure having this `Stop` call here was ever necessary. Was added during the last connectivity-improvement pr. The purpose was "ok if refreshing errors out, then just start over again completely by tearing down the mailboxes etc and basically forcing `NewServerConn` to be called again instead of Refresh being called again. But this was before the aperture issue was found. Since that was fixed, i havent yet run into a situation where this stop needs to be called here.
But happy to add it in the API for good measure 🤷‍♀️ let me know